Source/Core/DolphinNoGUI/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+if(APPLE)
+	set(DOLPHIN_EXE_BASE Dolphin)
+else()
+	set(DOLPHIN_EXE_BASE dolphin-emu)
+endif()
+
+set(NOGUI_SRCS MainNoGUI.cpp)
+
+list(APPEND LIBS core uicommon cpp-optparse)
```

Comment:
Instead of appending to the variable, can you just add those in the target_link_libraries() call below so it becomes:
```
target_link_libraries(dolphin-nogui PRIVATE
  core
  uicommon
  cpp-optparse
  ${LIBS}
)
```